portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,7 @@
 
 package com.google.sps.servlets;
 import com.google.sps.data.Comment;
+import static com.google.sps.data.StringConstants.*;
```

Comment:
Do we need the `*`? The Java style guide doesn't like that.

Reference: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
fixed!